### PR TITLE
internal/upgrade-building-script

### PR DESCRIPTION
**Pull request by Paurikova2**
| Phases            | MM  |  MB  | MR |   JM | Total  |
|-----------------|----:|----:|-----:|-----:|-------:|
| ETA                  |  0  |    0 |     0 |      0 |        0 |
| Developing      |  0  |    0 |    0 |      0 |         0 |
| Review             |  0  |    0 |    0 |      0 |         0 |
| Total                |   -  |   -   |  -    |   -    |         0 |
| ETA est.             |      |       |       |         |         0 |
| ETA cust.           |   -  |   -  |   -   |   -     |        0 |
## Analysis
(Write here, if there is needed describe some specific problem. Erase it, when it is not needed.)
## Problems
(Write here, if some unexpected problems occur during solving issues. Erase it, when it is not needed.)

### CODE REVIEW

**Review by MajoBerger — CHANGES_REQUESTED** [review]
why not use already used env file? Just add
`set m_2 source=`

**Comment by Paurikova2** [comment]
> 

Because we don't want to do it together with build.script. Or do we want it?

File: scripts/run.delete.dspace.parent.bat [hunk 1]
```diff
@@ -0,0 +1,11 @@
+rem set those to your local paths in .gitignored file /envs/__dspace.parent.basic.bat.
```
[the remark this answers is not in the document]
Comment (Paurikova2) [reply scripts/run.delete.dspace.parent.bat:1]: if I understand, now there is no need to write the information there...

[the remark this answers is not in the document]
Comment (MajoBerger) [reply scripts/run.delete.dspace.parent.bat:1]: yes

**Comment by MajoBerger** [comment]
The way I understand it, in that file that already exists, there are paths to relevant places on each PC. 
All can be in one place I think. Now for example, there is double definition of dspace_source. 
If we continued this way, there would be many different .env files each containing almost the same.
It is certainly bad practice to define the same value multiple times on multiple different locations.

That's why I think there should be only one file with envs...

File: scripts/envs/__dspace.parent.basic.example.bat [hunk 2]
```diff
@@ -0,0 +1,3 @@
+rem Set those paths to relevant places in your computer, copy this file and rename the copy to "dspace.parent.basic.bat"
+set m2_source=%USERPROFILE%\.m2
+set dspace_source=C:\workspace\DSpace
```
Comment (MajoBerger) [inline scripts/envs/__dspace.parent.basic.example.bat:3]: We agreed to remove this file and use `__basic.example.bat` instead, no?

File: scripts/run.delete.dspace.parent.bat [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+rem you HAVE TO CREATE /envs/__dspace.parent.basic.bat with following variables in the same directory
+
+rem start of variables expected in /envs/__dspace.parent.basic.bat
+set m2_source=
+set dspace_source=
+
+call envs\__dspace.parent.basic.bat
```
Comment (MajoBerger) [inline scripts/run.delete.dspace.parent.bat:8]: same as above. use `__basic.example.bat`

**Review by MajoBerger — CHANGES_REQUESTED** [review]
Don't use `__dspace.parent.basic.bat` but `__basic.bat` for envs

**Review by MajoBerger — APPROVED** [review]

**Pull request merged**